Improve UMA docs (#92)
wso2-extensions/identity-oauth-uma: docs/README.md

# User Managed Access with WSO2 Identity Server

UMA 2.0 is a standard protocol built on top of OAuth 2.0, which enables party-to-party sharing. It gives the resource owners the ability to control who can access their protected resources from a centralized authorization server by creating authorization policies.

## Getting started

> * To get started with User Managed Access, Go to [configuring UMA 2.0](config.md).  

Reviewer: ```suggestion
* To get started with User Managed Access, see the instructions on [configuring UMA 2.0](config.md).  
```

# User Managed Access with WSO2 Identity Server

> UMA 2.0 is a standard protocol built on top of OAuth 2.0 which enables party-to-party sharing. It gives the resource owner the ability to control who can access his/her protected resources from a centralized authorization server by creating authorization policies.

Reviewer: ```suggestion
UMA 2.0 is a standard protocol built on top of OAuth 2.0, which enables party-to-party sharing. It gives the resource owners the ability to control who can access their protected resources from a centralized authorization server by creating authorization policies.
```

## Getting started

* You can download the connector and other required artifacts from the [WSO2 store](https://store.wso2.com/store/assets/isconnector/list).

> * To getting started with User Managed Access, Go to [configuring UMA 2.0](config.md).  

Reviewer: ```suggestion
* To get started with User Managed Access, Go to [configuring UMA 2.0](config.md).  
```
